presto-main/src/main/java/com/facebook/presto/operator/BigintGroupByHash.java

```diff
@@ -267,6 +272,13 @@ private void rehash()
         }
         int newCapacity = (int) newCapacityLong;
 
+        // An estimation of how much extra memory needed before allocating the hash table
+        if (!updateReservedMemory.apply(((long) (newCapacity - hashCapacity)) * (Long.BYTES + Integer.BYTES))) {
+            // reserved memory but has exceeded the limit
+            // TODO enable this
+            // return;
```

Comment:
`TODO` will be removed in a later patch. The reason is to make sure each commit can be a standalone patch event without a later commit to enable this. If we remove comments in this commit without handling yields in the operator, it will fail.